<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quizpage</class>
 <widget class="QDialog" name="quizpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="QuizWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QPushButton" name="nextQuestBtn">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>340</y>
         <width>200</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>411</width>
         <height>501</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Questions</string>
       </property>
       <widget class="QPlainTextEdit" name="questionArea">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>39</y>
          <width>387</width>
          <height>180</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>180</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: white; 
                color: black;             
                border: 2px solid white;   
                border-radius: 20px;         
                padding: 10px;            
                font-size: 14px;            
                font-family: Arial; </string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="option3">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>406</y>
          <width>391</width>
          <height>74</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
        </property>
        <property name="text">
         <string>Button</string>
        </property>
       </widget>
       <widget class="QPushButton" name="option1">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>232</y>
          <width>391</width>
          <height>74</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
        </property>
        <property name="text">
         <string>Button</string>
        </property>
       </widget>
       <widget class="QPushButton" name="option2">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>319</y>
          <width>391</width>
          <height>74</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
        </property>
        <property name="text">
         <string>Button</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>40</y>
         <width>251</width>
         <height>281</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Timer</string>
       </property>
       <widget class="QLabel" name="timer_lbl">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>02:00</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn6">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>220</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>150</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn2">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn5">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>220</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>51</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="level_lbl">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>10</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="nextQuestBtn_2">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>340</y>
         <width>200</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
       </property>
       <property name="text">
        <string>Finish</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
